packages/kbn-monaco/src/code_editor/constants.ts

```diff
@@ -7,7 +7,5 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-export const CODE_EDITOR_LIGHT_THEME_ID = 'codeEditorLightTheme';
-export const CODE_EDITOR_DARK_THEME_ID = 'codeEditorDarkTheme';
-export const CODE_EDITOR_LIGHT_THEME_TRANSPARENT_ID = 'codeEditorLightTransparentTheme';
-export const CODE_EDITOR_DARK_THEME_TRANSPARENT_ID = 'codeEditorDarkTransparentTheme';
```

Comment:
There's no need for light and dark variations of the same theme anymore, we resolve this automatically now